app/views/handlers/individual_response.py

```diff
@@ -68,8 +68,9 @@ def __init__(
             raise NotFound
 
     def _is_location_valid(self):
-        if not self.router.can_access_hub():
-            return False
+        self._list_name = self._schema.json.get("individual_response", {}).get(
+            "for_list"
+        )
```

Comment:
There is a method to get the IR list in questionnaire schema, we should use that.